src/com/workspace/utill/Logger.java

```diff
@@ -0,0 +1,21 @@
+package com.workspace.utill;
+
+public class Logger {
+    private static Logger logger = null;
+    private static LogWriter logWriter = null;
+    private Logger(){
+        logWriter = new LogWriter();
+    }
+    public static Logger getLogger(){
+        if(logger == null){
+            logger = new Logger();
+        }
+        return logger;
+    }
+    public static void error(String message){
```

Comment:
This singleton class should only be responsible for providing the instance of the logger. Please keep these functions in a separate file and let this Logger implement that class.

Reply:
what do you mean here?